app/assets/stylesheets/sufia/_collections.scss

```diff
@@ -79,3 +79,7 @@ form.per_page {
     padding-left: 0;
   }
 }
+
+.collection_form_visibility .form-group {
+  padding-left: 25px;
```

Comment:
This is a stop gap.  The problem is wider than this issue and applies to several styles and views.  See issue #1987